pkg/apis/psmdb/v1/psmdb_defaults.go

```diff
@@ -783,10 +783,10 @@ func (m *MultiAZ) reconcileOpts(cr *PerconaServerMongoDB) {
 }
 
 var affinityValidTopologyKeys = map[string]struct{}{
-	AffinityOff:                                {},
-	"kubernetes.io/hostname":                   {},
-	"failure-domain.beta.kubernetes.io/zone":   {},
-	"failure-domain.beta.kubernetes.io/region": {},
+	AffinityOff:                     {},
+	"kubernetes.io/hostname":        {},
+	"topology.kubernetes.io/zone":   {},
```

Comment:
Maybe we need to do it only for >= 1.16.0

Reply:
yes! you're correct! My bad

Reply:
fixed in a5b2605386057eaea943a1f2f3b0111a6497d71d